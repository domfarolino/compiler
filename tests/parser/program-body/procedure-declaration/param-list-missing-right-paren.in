program dominic is
  procedure domFn(string myArg in;
begin end program.
