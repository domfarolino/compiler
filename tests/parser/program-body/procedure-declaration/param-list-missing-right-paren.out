[1;31mLine 2: Expected ')' after parameter list in procedure header, but got: ';'
Line 2: Error parsing program body
[0m